jenkins/jobs/capm3-e2e-tests.pipeline

```diff
@@ -0,0 +1,105 @@
+import java.text.SimpleDateFormat
+import java.security.SecureRandom
+
+ci_git_credential_id = "metal3-jenkins-github-token"
+
+// 3 hours
+int TIMEOUT = 10800
+
+KEEP_TEST_ENV = (env.KEEP_TEST_ENV)
```

Comment:
Can you remove KEEP_TEST_ENV var from here? it is not used in dynamic worker case.